tests/testthat/line_breaks_and_other/line_break_fun_dec-in.R

```diff
@@ -0,0 +1,17 @@
+a <- function(x, #
+              y) {
+  x - 1
+}
+
+
+a <- function(x, #
+              y) #
+{
+  x
+}
+
+a <- function(x, #
+              y #
+            ) {
```

Comment:
I think we can still add a FIXME here, because this isn't what we'd truly like to see.